secure_message/application.py

```diff
@@ -75,10 +75,7 @@ def handle_exception(error):  # NOQA
 
 
 def _request_requires_authentication():
-    return request.endpoint is not None and \
-           'health' not in request.endpoint \
-           and request.endpoint != 'info' \
-           and request.method != 'OPTIONS'
+    return request.endpoint is not None and 'health' not in request.endpoint and request.endpoint != 'info'and request.method != 'OPTIONS'
```

Comment:
Needs a space in between `'info'and` and also 140 characters so would be over the 120 mark